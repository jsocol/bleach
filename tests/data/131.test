<div id="58"><b><script<b></b><alert(58)</script </b></b>//["'`-->]]>]</div>
