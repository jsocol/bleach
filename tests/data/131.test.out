<div id="58"><b></b>//["'`--&gt;]]&gt;]</div>
